update-5etools-images.js

```diff
@@ -25,16 +25,17 @@ function getScenes() {
   return scenes;
 }
 
+
 async function doActors(replaceUrl, newUrl, logging) {
   for (const a of game.actors) {
     let data = {}
-    if (a.data.img.startsWith(replaceUrl)) {
+    if (a.data.img && a.data.img.startsWith(replaceUrl)) {
```

Comment:
Thank you for the fix! Please change these to use the `?.` operator to simplify it like the following:
```
if (a.data?.img.startsWith(replaceUrl)) {
```

Reply:
Dammit, google didn't show anything when I googled `nil failing object browsing js`. Changing it rn.